jsk_arc2017_baxter/euslisp/lib/util.l

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env roseus
+
+(require "package://jsk_2016_01_baxter_apc/euslisp/jsk_2016_01_baxter_apc/util.l")
+
+(ros::load-ros-manifest "jsk_gui_msgs")
+(ros::load-ros-manifest "jsk_recognition_msgs")
+
+(defun opposite-arm (arm) (if (eq arm :larm) :rarm :larm))
+
+(defun get-bin-contents (bin)
+  (ros::get-param (format nil "/bin_contents/~A"
+                            (string-upcase (symbol2str bin)))))
+
+(defun wait-for-user-input-to-start (arm)
```

Comment:
I'm not sure why you want to place this function outside of the apc-interface.
This is only used in apc, so it is ok for us if we can do as below:

```
;; *ti* is task interface
(setq *ti* (instance jsk_arc2017_baxter::apc-interface :init))
(send *ti* :wait-for-user-input-to-start :larm)
;; do tasks
```
And not sure what is the difference between this function and `:need-to-wait-opposite-arm` in `apc-interface`.